python-coverage.el

```diff
@@ -469,5 +469,39 @@ The EVENT causes the overlays in BUFFER to get refreshed."
 
   (add-to-list 'flycheck-checkers 'python-coverage t))
 
+(after 'python-coverage
+  (define-minor-mode python-coverage-mode
```

Comment:
since this is highly flycheck specific, lets name this `python-coverage-flycheck-mode` instead. (same in `if` below)